index.html

```diff
@@ -208,5 +208,30 @@ <h4>Skills</h4>
                 </ul>
                 <a href="#"><h5>tyaps002@gmail.com</h5></a>
             </section>
+            
+            <section class="form-survey">
```

Comment:
Your design doesn't quite match the template design.

Your design:

<img width="169" alt="Screen Shot 2022-06-08 at 15 02 52" src="https://user-images.githubusercontent.com/82426682/172706669-f54233b2-450d-4d03-bef5-b803353b4288.png">

The template design:

<img width="141" alt="Screen Shot 2022-06-08 at 15 03 20" src="https://user-images.githubusercontent.com/82426682/172706739-4136fe1f-c061-4b59-9fe7-b649569f0d38.png">

You're missing the background image, the content is not aligned properly and the text button is too small, among other things. Please take a closer look at the template design and copy it as close as you can as it is a project requirement.

<img width="900" alt="Screen Shot 2022-06-08 at 15 04 33" src="https://user-images.githubusercontent.com/82426682/172706929-94164e96-95f2-43ef-bb81-8c851eb86c16.png">